src/clients/telegram/telegram.ts

```diff
@@ -0,0 +1,70 @@
+import TelegramBot from 'node-telegram-bot-api'
+import { Client } from '../client'
+import { Message } from '../../types/messages'
+
+export class TelegramClient extends Client {
+  private readonly bot: TelegramBot
+  private readonly chatId: string
+  private readonly parseMode: TelegramBot.ParseMode
+  private readonly disableNotification: boolean
+
+  constructor() {
+    super()
+
+    if (!process.env.TELEGRAM_BOT_TOKEN || !process.env.TELEGRAM_CHAT_ID) {
+      throw new Error(
+        'telegram client is not configured. please check your .env file (TELEGRAM_BOT_TOKEN and TELEGRAM_CHAT_ID are required)',
+      )
+    }
+
+    const token = process.env.TELEGRAM_BOT_TOKEN
+    this.chatId = process.env.TELEGRAM_CHAT_ID
+    this.parseMode =
+      (process.env.TELEGRAM_PARSE_MODE as TelegramBot.ParseMode) || 'HTML'
+    this.disableNotification =
+      process.env.TELEGRAM_DISABLE_NOTIFICATION === 'true'
+
+    // initialize bot without polling since we only send messages
+    this.bot = new TelegramBot(token, { polling: false })
+  }
+
+  override async sendMessage(message: Message): Promise<void> {
+    try {
+      await this.bot.sendMessage(
+        this.chatId,
+        this.formatMessage(message.body),
+        {
+          parse_mode: this.parseMode,
+          disable_notification: this.disableNotification,
+          disable_web_page_preview: true,
+        },
+      )
+    } catch (error) {
+      const errorMessage =
+        error instanceof Error ? error.message : 'Unknown error'
+      throw new Error(`Failed to send Telegram message: ${errorMessage}`)
+    }
+  }
```

Comment:
we don't need to explicitly catch here, since the *nudgeManager.ts* handles the catch logic.

minor: you could consider building the options object separately before passing it in; it helps with readability when working with SDKs or 3rd party APIs that take multiple options.

```suggestion
  override async sendMessage(message: Message): Promise<void> {
      // your options here
      const telegramOptions: TelegramBot.SendMessageOptions = {}

      await this.bot.sendMessage(
        this.chatId,
        this.formatMessage(message.body),
        telegramOptions,
      )
  }
```